kubetest/anywhere.go

```diff
@@ -46,6 +46,10 @@ var (
 		"(kubernetes-anywhere only) Time limit between starting a cluster and making a successful call to the Kubernetes API.")
 	kubernetesAnywhereNumNodes = flag.Int("kubernetes-anywhere-num-nodes", 4,
 		"(kubernetes-anywhere only) Number of nodes to be deployed in the cluster.")
+	kubernetesAnywhereUpgradeProvider = flag.String("kubernetes-anywhere-upgrade-provider", "kubeadm",
+		"(kubernetes-anywhere only) The upgrade provider")
+	kubernetesAnywhereKubeadmUpgradeMethod = flag.String("kubernetes-anywhere-kubeadm-upgrade-method", "upgrade",
```

Comment:
can we rename this to the more generic `kubernetes-anywhere-upgrade-method` and use `kubeadm-init` and `kubeadm-upgrade` instead?

Reply:
That is probably fine as long as the right options are set in the kubernetes anywhere configuration.